plugins/contact-resources/src/components/ChannelsDropdown.svelte

```diff
@@ -203,32 +211,53 @@
     })
   }
   let copied: boolean = false
+  let div: HTMLDivElement
+
+  function listener (e: MouseEvent): void {
+    if (e.target !== null && !div.contains(e.target as Node)) {
+      disableEdit()
+    }
+  }
+
+  function enableEdit () {
+    window.addEventListener('click', listener)
+    editMode = true
+  }
+
+  function disableEdit () {
+    window.removeEventListener('click', listener)
```

Comment:
Please use onDestroy to force remove listener or <svelte:window on:click> and inner handler.